src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -55,13 +46,24 @@ const Payments = ({ data, fields, tableVersion, textContent }) => {
         <p className="vads-u-font-size--lg vads-u-font-family--serif">
           Displaying {fromToNums[0]} - {fromToNums[1]} of {data.length}
         </p>
-        <Table
-          ariaLabelledBy={tableAriaLabelledBy}
-          className="va-table"
-          fields={fields}
-          data={currentData}
-          maxRows={MAX_ROWS}
-        />
+        <va-table>
+          <va-table-row slot="headers">
+            {fields.map(field => (
+              <span key={field.value}>{field.label}</span>
```

Comment:
I saw [the note about why a `span` is used instead of table cells](https://github.com/department-of-veterans-affairs/component-library/pull/291#pullrequestreview-892879766), but why is a `va-table-row` needed in the first place? What happened to the `thead`, `tbody` and `tfoot`?

I haven't seen any examples, but what if someone wants to add a sticky header or footer?

Reply:
> I saw https://github.com/department-of-veterans-affairs/component-library/pull/291#pullrequestreview-892879766, but why is a va-table-row needed in the first place? What happened to the thead, tbody and tfoot?

> I haven't seen any examples, but what if someone wants to add a sticky header or footer?

I don't have additional context about the architectural decisions of the web component besides what you've also linked to so I don't think I can provide any more clarification about that.

But I do think that when this component eventually gets transitioned to the [USWDS v3 version](https://designsystem.digital.gov/components/table/), we will need to reevaluate what elements it's rendering because they do include thead, tr, th, etc (which I think we need to also maintain). At the moment though, we are simply trying to deprecate the React Table component.